src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -553,5 +554,6 @@ void DlgPrefLibrary::updateSearchLineEditHistoryOptions() {
 void DlgPrefLibrary::slotSyncTrackMetadataToggled() {
     if (isVisible() && checkBox_SyncTrackMetadata->isChecked()) {
         mixxx::DlgTrackMetadataExport::showMessageBoxOncePerSession();
+        checkBox_SeratoMetadataExport->setEnabled(checkBox_SyncTrackMetadata->isChecked());
```

Comment:
Why is this in this if branch? I also do not understand what the "isVisible()" check is for. 
Does this guard against the initial call? 

You can debug it by placing qDebug() statements in this function. They are printed into the console if you start mixxx with "./mixxx --developer"

Reply:
Your question is valid. But unrelated and does not matter in the context of this PR.

Reply:
For my understanding this is wrong.

Reply:
Correct me if I'm wrong, but IIUC this prevents the message box to be shown when the checkbox is updated (e.g. in `slotUpdate`?) while DlgPrefLibrary is not shown, e.g. during consctruction.

Reply:
Please stop discussing unrelated questions in a PR. Thank you.

Reply:
Why do you think this is unrelated. The code is wrong, we need to set the state unconditionally.

Reply:
@tech-bash Please fix your code as originally suggested.

Reply:
I will open a seperate PR.